algos.yaml

```diff
@@ -58,6 +58,18 @@ float:
         base:
           args: [[400, 1024, 4096, 8192, 16384],
                  [1, 10, 40, 100, 200]]
+    raft-ivfpq:
+      disabled: true
```

Comment:
why is it disabled?

Reply:
Only because it requires a GPU and I wasn't sure if it's okay to assume one will always be available.

Reply:
ok that makes sense! yeah let's keep it disabled then